curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-drum-machine.md

```diff
@@ -28,6 +28,26 @@ You can use any mix of HTML, JavaScript, CSS, Bootstrap, SASS, React, Redux, and
 
 **User Story #7:** When a `.drum-pad` is triggered, a string describing the associated audio clip is displayed as the inner text of the `#display` element (each string must be unique).
 
+**Use the following audio clips for your project:**
+
+**Heater-1** `Q` <a href='https://s3.amazonaws.com/freecodecamp/drums/Heater-1.mp3'>https://s3.amazonaws.com/freecodecamp/drums/Heater-1.mp3</a>
+
+**Heater-2** `W` <a href='https://s3.amazonaws.com/freecodecamp/drums/Heater-2.mp3'>https://s3.amazonaws.com/freecodecamp/drums/Heater-2.mp3</a>
+
+**Heater-3** `E` <a href='https://s3.amazonaws.com/freecodecamp/drums/Heater-3.mp3'>https://s3.amazonaws.com/freecodecamp/drums/Heater-3.mp3</a>
+
+**Heater-4** `A` <a href='https://s3.amazonaws.com/freecodecamp/drums/Heater-4_1.mp3'>https://s3.amazonaws.com/freecodecamp/drums/Heater-4_1.mp3</a>
+
+**Clap** `S` <a href='https://s3.amazonaws.com/freecodecamp/drums/Heater-6.mp3'>https://s3.amazonaws.com/freecodecamp/drums/Heater-6.mp3</a>
+
+**Open-HH** `D` <a href='https://s3.amazonaws.com/freecodecamp/drums/Dsc_Oh.mp3'>https://s3.amazonaws.com/freecodecamp/drums/Dsc_Oh.mp3</a>
+
+**Kick-n'-Hat** `Z` <a href='https://s3.amazonaws.com/freecodecamp/drums/Kick_n_Hat.mp3'>https://s3.amazonaws.com/freecodecamp/drums/Kick_n_Hat.mp3</a>
+
+**Kick** `X` <a href='https://s3.amazonaws.com/freecodecamp/drums/RP4_KICK_1.mp3'>https://s3.amazonaws.com/freecodecamp/drums/RP4_KICK_1.mp3</a>
+
+**Closed-HH** `C` <a href='https://s3.amazonaws.com/freecodecamp/drums/Cev_H2.mp3'>https://s3.amazonaws.com/freecodecamp/drums/Cev_H2.mp3</a>
```

Comment:
Rather than using a bunch of space and raw links, I think we should use a proper Markdown list:

```suggestion
- [Heater 1](https://s3.amazonaws.com/freecodecamp/drums/Heater-1.mp3)
- [Heater 2](https://s3.amazonaws.com/freecodecamp/drums/Heater-2.mp3)
- [Heater 3](https://s3.amazonaws.com/freecodecamp/drums/Heater-3.mp3)
- [Heater 4](https://s3.amazonaws.com/freecodecamp/drums/Heater-4_1.mp3)
- [Clap](https://s3.amazonaws.com/freecodecamp/drums/Heater-6.mp3)
- [Open-HH](https://s3.amazonaws.com/freecodecamp/drums/Dsc_Oh.mp3)
- [Kick-n'-Hat](https://s3.amazonaws.com/freecodecamp/drums/Kick_n_Hat.mp3)
- [Kick](https://s3.amazonaws.com/freecodecamp/drums/RP4_KICK_1.mp3)
- [Closed-HH](https://s3.amazonaws.com/freecodecamp/drums/Cev_H2.mp3)
```

Additionally, I think we should remove the reference to the letter keys here - campers are free to assign whichever sounds they'd like to their keys.

Reply:
Thx <3